cast/src/starknet_commands/helpers.rs

```diff
@@ -0,0 +1,10 @@
+use starknet::core::types::FieldElement;
+use anyhow::Result;
+
+pub fn parse_number(contract_address_arg: &str) -> Result<FieldElement> {
```

Comment:
Since this function is universal let's change the `contract_address_` prefix to something more suitable

Reply:
[address fdbck](https://github.com/software-mansion/protostar/pull/2126/commits/292f09298a7bc2981fd4bf4a0a4c980ec2ab6bed)